src/utils/helpers/currencies.ts

```diff
@@ -1,9 +1,20 @@
 import { CurrencyExt } from '@interlay/interbtc-api';
+import { MonetaryAmount } from '@interlay/monetary-js';
 
 // Squid query by currency ticker for native assets and by id for foreign assets.
 // We need to differentiate because those are handled differently on squid side.
 // TODO: Need to refactor when we want to support lend tokens as collateral for vaults.
 const getCurrencyEqualityCondition = (currency: CurrencyExt): string =>
   'foreignAsset' in currency ? `asset_eq: ${currency.foreignAsset.id}` : `token_eq: ${currency.ticker}`;
 
-export { getCurrencyEqualityCondition };
+const pickSmallerAmount = (
```

Comment:
`pickAmount(type: "lowest" | "highest", amount0...,amount1...)`

What do you think?

Reply:
I think it's unnecessary abstraction for this case - seems more clear to me to see what function is doing without having to pass in another parameter.

Reply:
I said this because I us using this in another places and you have placed this in a common currency helper. Leave it like this is fine :)